optical/converter/__init__.py

```diff
@@ -48,12 +50,6 @@ def bbox_stats(self, split: Optional[str] = None, category: Optional[str] = None
     def convert(self, to: str, output_dir: Optional[Union[str, os.PathLike]] = None, **kwargs):
         if not to.lower() in SUPPORTED_FORMATS:
             raise ValueError(f"`{format}` is not a supported conversion format")
-
```

Comment:
Please leave the commented out code part, don't delete them! we don't want the user to convert between same source and destination format. I was just testing so i had commented that.